app/admin/subsite.rb

```diff
@@ -0,0 +1,27 @@
+ActiveAdmin.register Subsite do
+  permit_params :name, :site_id
+
+  index do
+    column :name
+    column :site do |s|
+      "#{s.property&.name} - #{s.name}"
```

Comment:
I'm having trouble with this on my local dev environment because `s` here gets set to the subsite, and so `s.property` doesn't work. Is there an easy fix for this?